src/library/simplified/extension/marketing-ab-tests.js

```diff
@@ -0,0 +1,16 @@
+utag_data['ABTestsExpValues'] = '';
+var tpid = utag_data['context.site.tpid'];
+var eapid = utag_data['context.site.eapid'];
+var guid = utag_data['guid'];
+var domain_name = utag_data['context.site.siteName'];
+var expId = utag_data['marketingABTests'];
+var xhttp = new XMLHttpRequest();
+xhttp.open('get', 'https://'+domain_name+'/api/bucketing/v1/evaluateExperimentsAndLog?guid='+guid+'&tpid='+tpid+'&eapid='+eapid+'&id='+expId+'', false);
+xhttp.send();
```

Comment:
could you please use https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/timeout